cloud2cloud-connector/service/getDevices_test.go

```diff
@@ -18,7 +18,7 @@ import (
 	kitNetGrpc "github.com/plgd-dev/hub/v2/pkg/net/grpc"
 	"github.com/plgd-dev/hub/v2/resource-aggregate/commands"
 	"github.com/plgd-dev/hub/v2/test"
-	testCfg "github.com/plgd-dev/hub/v2/test/config"
+	config "github.com/plgd-dev/hub/v2/test/config"
```

Comment:
`config "github.com/plgd-dev/hub/v2/test/config"` can be shortened to just `"github.com/plgd-dev/hub/v2/test/config"`